pkg/common/helper/helper.go

```diff
@@ -26,67 +29,128 @@ func init() {
 
 // New creates H, a helper used to expose common testing functions.
 func New() *H {
-	helper := &H{
+	h := &H{
 		State: state.Instance,
 	}
 
-	ginkgo.BeforeEach(helper.Setup)
-	ginkgo.AfterEach(helper.Cleanup)
-	return helper
+	ginkgo.BeforeEach(h.Setup)
+	return h
 }
 
 // H configures clients and sets up and destroys Projects for test isolation.
 type H struct {
 	// embed state
 	*state.State
+	Persona string
 
 	// internal
 	restConfig *rest.Config
 	proj       *projectv1.Project
 }
 
-// SetupNoProj configures a *rest.Config using the embedded kubeconfig
-func (h *H) SetupNoProj() {
+// Setup configures a *rest.Config using the embedded kubeconfig then sets up a Project for tests to run in.
+func (h *H) Setup() {
 	var err error
 
+	h.restConfig, err = clientcmd.RESTConfigFromKubeConfig(h.Kubeconfig.Contents)
+	Expect(err).ShouldNot(HaveOccurred(), "failed to configure client")
+
+	if config.Instance.Tests.Persona != "" {
+		h.Persona = config.Instance.Tests.Persona
+	}
+
+	if h.State.Project == "" {
+		// setup project and dedicated-admin account to run tests
+		// the service account is provisioned but only used when specified
+		suffix := util.RandomStr(5)
+		h.State.Project = "osde2e-" + suffix
+
+		log.Printf("Setup called for %s", h.State.Project)
+
+		sa, err := h.Kube().CoreV1().ServiceAccounts("dedicated-admin").Create(&v1.ServiceAccount{
```

Comment:
Yes, we create one per-project. No, by default we do not run each test as dedicated-admin. We can set a global persona to run all helper-based tests as, or we can set it per-test (privileged container test is an example where we set it for the test specifically)

Since we're cutting down on the number of projects we create, and because now (and for the foreseeable future, we're only ever testing one of two personas) I pre-allocate the dedicated-admin SA since their lifecycles are intertwined.